node/resolvers/auth/index.ts

```diff
@@ -28,9 +28,10 @@ export async function makeRequest({
   cookie = null,
 }: ParamsMakeRequest) {
   const composedHeaders = {
-    ...authHeaders.profile,
     Cookie: `VtexIdClientAutCookie=${cookie}`,
     'vtex-ui-id-version': vtexIdVersion,
+    accept: 'application/vnd.vtex.ds.v10+json',
+    ...(body && { 'content-type': 'application/x-www-form-urlencoded' }),
```

Comment:
`Content-Type` used to be `json`, but there was no body anyway. This function will now sometimes be used with a `urlencoded` body.